Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Dashboard/Operator/Driver.hs

```diff
@@ -119,3 +128,43 @@ castReqType :: RequestType -> API.Types.ProviderPlatform.Operator.Driver.Request
 castReqType = \case
   ONBOARDING_INSPECTION -> API.Types.ProviderPlatform.Operator.Driver.ONBOARDING_INSPECTION
   REGULAR_INSPECTION -> API.Types.ProviderPlatform.Operator.Driver.REGULAR_INSPECTION
+
+getDriverOperatorList ::
+  Kernel.Types.Id.ShortId Domain.Types.Merchant.Merchant ->
+  Kernel.Types.Beckn.Context.City ->
+  Kernel.Prelude.Maybe Kernel.Prelude.Bool ->
+  Kernel.Prelude.Maybe Kernel.Prelude.Bool ->
+  Kernel.Prelude.Maybe Kernel.Prelude.Int ->
+  Kernel.Prelude.Maybe Kernel.Prelude.Int ->
+  Kernel.Prelude.Text ->
+  Environment.Flow [API.Types.ProviderPlatform.Operator.Driver.DriverInfo]
+getDriverOperatorList _merchantShortId _opCity mbIsActive mbVerified mbLimit mbOffset requestorId = do
+  driverOperatorAssociationLs <-
+    findAllByOperatorIdWithLimitOffset requestorId mbIsActive mbLimit mbOffset
+  mapM createDriverInfo driverOperatorAssociationLs
+  where
+    createDriverInfo drvOpAsn = do
+      let driverId = drvOpAsn.driverId
+      person <- QPerson.findById driverId >>= fromMaybeM (PersonNotFound driverId.getId)
+      decryptedMobileNumber <-
+        mapM decrypt person.mobileNumber
+          >>= fromMaybeM
+            ( InvalidRequest $
+                "Person do not have a mobile number " <> person.id.getId
+            )
+      mblinkedVehicle <- QVehicle.findById driverId
+      driverInformation <-
+        QDI.findByIdAndVerified driverId mbVerified
+          >>= fromMaybeM DriverInfoNotFound
```

Comment:
why you throw error here?
First you fetch all associations, not using this `mbVerified` flag
`driverOperatorAssociationLs <- findAllByOperatorIdWithLimitOffset requestorId mbIsActive mbLimit mbOffset`
So if `isJust mbVerified` and at least one `driverInfo.verified` is not the same as `mbVerified` then error will be thrown right?

Reply:
If you just skip value with wrong `mbVerified`, then entities number may be less than `limit`

@0utkarsh should we use join query here, to fetch list of both association and `driverInfo` with specified `mbVerified` flag? What do you think?